macros/CSSRef.ejs

```diff
@@ -17,12 +17,12 @@ if (slug) {
     var htmlEscape = kuma.htmlEscape;
     var rtlLocales = ['ar', 'he', 'fa'];
 
-    var propertiesUrl = "https://raw.githubusercontent.com/mdn/data/mdn/css/properties.json";
-    var atRulesUrl = "https://raw.githubusercontent.com/mdn/data/mdn/css/at-rules.json";
-    var selectorsUrl = "https://raw.githubusercontent.com/mdn/data/mdn/css/selectors.json";
-    var typesUrl = "https://raw.githubusercontent.com/mdn/data/mdn/css/types.json";
-    var syntaxesUrl = "https://raw.githubusercontent.com/mdn/data/mdn/css/syntaxes.json";
-    var unitsUrl = "https://raw.githubusercontent.com/mdn/data/mdn/css/units.json";
+    var propertiesUrl = "https://raw.githubusercontent.com/mdn/data/master/css/properties.json";
+    var atRulesUrl = "https://raw.githubusercontent.com/mdn/data/master/css/at-rules.json";
+    var selectorsUrl = "https://raw.githubusercontent.com/mdn/data/master/css/selectors.json";
+    var typesUrl = "https://raw.githubusercontent.com/mdn/data/master/css/types.json";
+    var syntaxesUrl = "https://raw.githubusercontent.com/mdn/data/master/css/syntaxes.json";
+    var unitsUrl = "https://raw.githubusercontent.com/mdn/data/master/css/units.json";
```

Comment:
There's a bug in ``pages.subpagesExpand()`` and similar page macros.  Will opened [bug 1368134](https://bugzilla.mozilla.org/show_bug.cgi?id=1368134), and that should cover the work.  If I alter ``DekiScript-Page.ejs`` to use ``api:8000`` vs ``p.host``, the child call works and the sidebar is populated.